topology/University of Campinas/CCJDR/feynman-PBS.yaml

```diff
@@ -0,0 +1,119 @@
+# Production is true if the resource is for production and not testing use
+Production: false
```

Comment:
Is this registration for a test CE?

Reply:
@biozit is this for a test or production CE?

Reply:
yes, it is.